src/config/config.ts

```diff
@@ -21,12 +21,12 @@ export const apiConfig = {
         process.env.PLATFORM_API_BASE_URL ||
         "https://platform.nypl.org/api/v0.1",
     },
-    drbb: {
+    drb: {
       development:
         process.env.DRB_API_BASE_URL || "http://drb-api-qa.nypl.org/search/",
       production:
         process.env.DRB_API_BASE_URL ||
-        "https://digital-research-books-api.nypl.org/v3/sfr/search",
+        "https://digital-research-books-api.nypl.org/search",
```

Comment:
Who verified this is the stable version?

Reply:
This url is what's being used on production currently. The old fallback url is for v3.